modules/virt-overriding-default-fs-overhead-value.adoc

```diff
@@ -6,43 +6,42 @@
 [id="virt-overriding-default-fs-overhead-value_{context}"]
 = Overriding the default file system overhead value
 
-Change the amount of persistent volume claim (PVC) space that the Containerized Data Importer (CDI) reserves for file system overhead by editing the `spec.config.filesystemOverhead` attribute of the `CDI` object.
+Change the amount of persistent volume claim (PVC) space that OpenShift Virtualization reserves for file system overhead by editing the `spec.filesystemOverhead` attribute of the `HCO` object.
 
 .Prerequisites
 
 * Install the OpenShift CLI (`oc`).
 
 .Procedure
 
-. Open the `CDI` object for editing by running the following command:
+. Open the `HCO` object for editing by running the following command:
 +
 [source,terminal]
 ----
-$ oc edit cdi
+$ oc edit hco
 ----
 
-. Edit the `spec.config.filesystemOverhead` fields, populating them with your chosen values:
+. Edit the `spec.filesystemOverhead` fields. You can choose from either or both a global value that will apply to all storage classes, or a storage class specific value.
```

Comment:
You can choose from either or both a global value that will apply to all storage classes, or a storage class specific value. --> Changed to:
  You can choose to populate either a global value that will apply to all storage classes, or a storage class specific value or both.